### PR TITLE
5 api/articles/article

### PR DESCRIPTION
**Pull request by druoc**

### CODE REVIEW

**Review by Poonam-raj — CHANGES_REQUESTED** [review]
Hey 👋🏾 
I need you to take out the code relating to GEt 'api/articles' from this branch as this branch is entirely related to 'api/articles/:article_id' 

I also need you to take the .only off your testing and resubmit this PR when you've done that thank you 😁

File: __tests__/app.test.js [hunk 1]
```diff
@@ -43,3 +43,39 @@ describe('GET/api/topics', () => {
...
+						topic: expect.any(String),
+						author: expect.any(String),
+						body: expect.any(String),
+						created_at: expect.any(String),
+						votes: expect.any(Number),
+					})
+				);
+			});
+	});
+	test('responds with a 404 status code an an error object when passed an invalid article id', () => {
```
Comment (k-huggs) [inline __tests__/app.test.js:67]: missing a 400 test for invalid endpoint

File: controllers/news.js [hunk 2]
```diff
@@ -1,8 +1,20 @@
...
 		res.status(200).send(returnedTopics);
-		console.log({ topics: returnedTopics });
+	});
+};
+
+exports.getArticlesById = (req, res) => {
+	const { article_id } = req.params;
+	selectArticlesById(article_id).then((returnedArticle) => {
+		returnedArticle === undefined
+			? res.status(404).send({ msg: 'Route not found' })
```
Comment (k-huggs) [inline controllers/news.js:17]: route not found may be slightly misleading as a error message, you also don't want to be doing this in our controller

**Review by k-huggs — COMMENTED** [review]
Hi mate, have a look at the comments, you want the controller should just be handling the request and response. Other functionality should be coming from your model

**Pull request merged**